modules/openapi-generator/src/main/resources/php/composer.mustache

```diff
@@ -15,12 +15,12 @@
         "rest",
         "api"
     ],
-    "homepage": "https://openapi-generator.tech",
-    "license": "unlicense",
+    "homepage": "{{artifactUrl}}",
```

Comment:
please use `{{{ ... }}}` instead of `{{ ... }}`, which outputs HTML escaped value.

Reply:
will take care of it after merging this PR into master